File: src/main/kotlin/ru/andreyTw/delivery/repository/ClientTypeDataRepository.java
package ru.andreyTw.delivery.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import ru.andreyTw.delivery.db.ClientTypeData;

@Repository
public interface ClientTypeDataRepository extends JpaRepository<ClientTypeData, Long> {
}

File: src/main/kotlin/ru/andreyTw/delivery/repository/ClientTypeDataRepository.kt
package ru.andreyTw.delivery.repository

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import ru.andreyTw.delivery.db.ClientTypeData

@Repository
interface ClientTypeDataRepository : CrudRepository<ClientTypeData?, Long?> {
    fun findByName(name: String?): ClientTypeData?
}
